themes/digital.gov/src/scss/new/_in-page-navigation.scss

```diff
@@ -0,0 +1,17 @@
+@use "uswds-core" as *;
+
+//overrides for in-page nav component on /resources
+.usa-in-page-nav {
+  // sets in-page column to be even with content
+  margin-top: 0;
+
+  &__nav {
+    padding-left: 0;
+    padding-top: units('05');
+    margin-bottom: units(4);
+  }
+
+  &__item--sub-item {
+    display: none;
+  }
+}
```

Comment:
Should this be in `_uswds-overrides.scss`?

Reply:
I see your point, moved to `_uswds-overrides.scss`.

Reply:
Let's move all the overriding styles to the overrides partial and delete this file.